src/lsp/cobol_cfg/cobol_cfg.ml

```diff
@@ -0,0 +1,23 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+module INTERNAL = struct
+  module Types = struct
+  end
+end
+
+module Builder = struct
```

Comment:
I think it will make more sense to add a stripped-down `cfg_builder.mli` and then only have `module Builder = Cfg_builder`.